src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing Cloud Credentials, a central location to manage your authentication with AWS and GCP  <!-- 8550 -->
+- Enabling Okteto Insights for all users of Okteto <!-- 8627 -->
+- Added configurable buildkit network mode  <!-- 8486 -->
```

Comment:
Should we link to [here](https://www.okteto.com/docs/1.25/self-hosted/helm-configuration/#buildkit)?

Reply:
This was already shipped in the previous version, right?

Reply:
Yes, this was released on previous version, so we could skip it